RawVideo/VideoFrameRenderer.cs

```diff
@@ -57,17 +53,19 @@ public unsafe void RenderRawVideoFrame(RawVideoFrameBuffer videoFrameBuffer)
                 destArrayBuffer[i + 3] = sourceArrayBuffer[i + 3];
```

Comment:
After copy the bytes, should we free sourceArrayBuffer?

Reply:
sourceArrayBuffer and destArrayBuffer are owned by the respective IBuffers if you delete the memory it will crash, IBuffer will take care of deleting the memory allocated, also i saw the GC working as expected